proto/stride/auction/bid.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package stride.auction;
+
+import "google/protobuf/any.proto";
+import "gogoproto/gogo.proto";
+
+
+option go_package = "github.com/Stride-Labs/stride/v5/x/auction/types";
+
+/* ****************************************************************************
+An Order is a pair of price and volume
+A Bid is one or more orders
+
+A SealedBid is who sent it and the hash of the bid + secretSalt
+An OpenBid is who sent it with the Bid data structure
+**************************************************************************** */
+
+message Order {
+    uint64 price = 1 [
```

Comment:
The type of price field must be always integer? I think it should be decimal type more generally.
What do you think?

Reply:
This was something I was unsure about because different denom have different subunits.  I asked about this and was told most are 10^6 so 1ATOM is represented as 1000000 but some might be 10^12 or 10^18 subunits just depending on the denom.  If you think the sdk DEC is a better way to represent this or if that is what we do in other parts of the code base I'm happy to make this change.

Reply:
I see. Lets keep it as it is now.